tests/Wrap.js

```diff
@@ -46,4 +46,21 @@ describe('<Wrap /> Check id`s to render the SVG', () => {
   it('has `stop` inside the `linearGradient`', () => {
     expect(wrapper.find('linearGradient').find('stop')).to.have.length(3)
   })
+
+  it('has there `animate`', () => {
+    expect(wrapper.find('animate')).to.have.length(3)
+  })
+
+  it('has one `animate` inside each `stop`', () => {
+    const stopElem = wrapper.find('stop')
+    expect(stopElem.map(elem => elem.find('animate'))).to.have.length(3)
+  })
+})
+
+describe('<Wrap /> Check the case when animate={false}', () => {
```

Comment:
Put this test together with other test relate `animate` props.
Use only the `index.js|<ContentLoader />` to test the props

Reply:
This test verify if there is no animate element inside `<Wrap />`. I'm already testing the prop on `index.js|<ContentLoader />`, then this test should be removed?